commons.py

```diff
@@ -17,16 +17,13 @@ def create_table(con: sqlite3.Connection):
 
 
 def get_random_area_code() -> str:
-    return ''.join([F"{random.randint(0, 9)}" for _ in range(0, 6)])
-
+    return str(random.randint(100000, 999999))
 
 def get_random_age() -> int:
-    return random.choice([5, 10, 15])
-
+    return random.choice(range(5,16,5))
```

Comment:
Any chance you know why this is faster than the simpler implementation?

Reply:
I only tried because I noticed the random.sample doc mentioning it:

>To choose a sample from a range of integers, use a range() object as an argument. This is especially fast and space efficient for sampling from a large population: sample(range(10000000), k=60).

Turns out, it was faster with random.choice as well, since they share some common internals.

Reply:
Added a comment